pyproject.toml

```diff
@@ -82,7 +83,7 @@ vllm_tgis_adapter = [
 ]
 
 [tool.pytest.ini_options]
-addopts = "-ra -W error::grpc.experimental.ExperimentalApiWarning"
+addopts = "-ra"
```

Comment:
`pytest -W` matches the behaviour of the python cli:

```
The full form of argument is: action:message:category:module:lineno
```

This caused `grpc.experimental.ExperimentalApiWarning` to be raised as an exception when running the healthcheck against the grpc server. We really don't need this.